### DISCUSSION TITLE
Welcome to bamoe-docs Discussions!

### DISCUSSION THREAD

**Discussion by gitgabrio** [IBM/bamoe-docs#132] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gitgabrio** [comment]
Hi @LightGuard @justlmills @jomarko @yesamer @porcelli @baldimir 
Looking at documentation, I see lot of ambiguity about the "Decision" term. It is used for both "DMN" and "DRL" models, and to me this seems confusing.
I also see from some community requests that people uses the term "decision" when talking of drl, and funny enough someone calls "rules" the DMN decisions.
I think we should clearly separate and identify them, using "Decision" for DMN and "Rules" for DRL.
But probably there is some other reason behind: wdyt ?

**Reply by jomarko** [reply]
I agree we can try to be consistent in the naming on our side, in the bamoe docs. I agree, we can use Decision as synonym for DMN and Rule as synonym for DRL. However, we should be very tolerant in tickets from the users and customers, if they use it differently and rather just double check: "Is your decision/rule DMN or DRL based?"

**Comment by LightGuard** [comment]
Works for me. That was the distinction I’d been using as well.
